modules/stanford_soe_helper_collection_views/stanford_soe_helper_collection_views.views_default.inc

```diff
@@ -0,0 +1,1393 @@
+<?php
+/**
+ * @file
+ * stanford_soe_helper_collection_views.views_default.inc
+ */
+
+/**
+ * Implements hook_views_default_views().
+ */
+function stanford_soe_helper_collection_views_views_default_views() {
+  $export = array();
+
+  $view = new view();
+  $view->name = 'stanford_magazine_collection_mag_landing_page';
+  $view->description = '';
+  $view->tag = 'default';
+  $view->base_table = 'node';
+  $view->human_name = 'Stanford Magazine Collection: Mag. Landing Page';
+  $view->core = 7;
+  $view->api_version = '3.0';
+  $view->disabled = FALSE; /* Edit this to true to make a default view disabled initially */
+
+  /* Display: Master */
+  $handler = $view->new_display('default', 'Master', 'default');
+  $handler->display->display_options['css_class'] = 'views-grid-three article-grouping';
+  $handler->display->display_options['use_more_always'] = FALSE;
+  $handler->display->display_options['access']['type'] = 'perm';
+  $handler->display->display_options['cache']['type'] = 'time';
+  $handler->display->display_options['cache']['results_lifespan'] = '3600';
+  $handler->display->display_options['cache']['results_lifespan_custom'] = '0';
+  $handler->display->display_options['cache']['output_lifespan'] = '3600';
+  $handler->display->display_options['cache']['output_lifespan_custom'] = '0';
+  $handler->display->display_options['query']['type'] = 'views_query';
+  $handler->display->display_options['query']['options']['disable_sql_rewrite'] = TRUE;
```

Comment:
Why is this disabled? Doesn't this then cause this view to bypass the access and permissions hooks when fetching content?